OpenRA.Mods.Common/DiscordService.cs

```diff
@@ -189,18 +195,8 @@ void SetStatus(DiscordState state, string details = null, string secret = null,
 
 		void UpdateParty(int players, int slots)
 		{
-			if (client.CurrentPresence.Party != null)
-			{
+			if (client.CurrentPresence.HasParty())
 				client.UpdatePartySize(players, slots);
-				return;
-			}
-
-			client.UpdateParty(new Party
-			{
-				ID = Secrets.CreateFriendlySecret(new Random()),
-				Size = players,
-				Max = slots
-			});
```

Comment:
Why do you call this dead code?

Reply:
I guess it no longer crashes? 1a77f7320bf3ddd2c0a9cb44ed10aad77dbeb840

Reply:
In my tests this part of the code is never hit, but I better keep it then.